deploy/trustification.dev/staging.yaml

```diff
@@ -164,6 +166,7 @@ v11y:
       stored: v11y-stored-staging
       indexed: v11y-indexed-staging
       failed: v11y-failed-staging
+    indexSyncInterval: 5m
```

Comment:
The v11y index is pretty big though I think, might want to use a higher interval.